### ISSUE TITLE
Mysql connection dies

### ISSUE THREAD

**Issue by ddeath** [BitcoinExchangeFH/BitcoinExchangeFH#122]
Hi when I am using this app with mysql, mysql dies. 

I guess it is due to multithreading because some of the sql errors are:
[MysqlClient]
SQL error: Packet sequence number wrong - got 1 expected 2

Also in https://github.com/PyMySQL/mysqlclient-python/blob/master/doc/user_guide.rst : 
```
The general upshot of this is: Don't share connections between threads. It's really not worth your effort or mine, and in the end, will probably hurt performance, since the MySQL server runs a separate thread for each connection. You can certainly do things like cache connections in a pool, and give those connections to one thread at a time. If you let two threads use a connection simultaneously, the MySQL client library will probably upchuck and die. You have been warned.
```

Also pymysql states that the error above occurs when the lib is used in the incorrect way:
https://github.com/PyMySQL/PyMySQL/issues/425

**Comment by gavincyi** [comment]
Yeah I have a plan to redesign the whole stuff and no longer use threading. To workaround, could you provide me your pymysql version?

**Comment by ddeath** [comment]
Yop, I am using version 0.8 I found out that it looks like the problem is only during the creation of tables.
It runs fine all day and then after midnight if freezes (new day new tables) I have to restart process multiple time until I got lucky and tables are created without freeze...

**Comment by gavincyi** [comment]
Resolved in 2019.1.1
